storage/google/cloud/storage/blob.py

```diff
@@ -586,6 +588,35 @@ def _do_download(
             while not download.finished:
                 download.consume_next_chunk(transport)
 
+    def get_streaming_file(self, filename, client=None):
```

Comment:
This method is similar to `download_to_filename()`, but it doesn't use `start` and `end` args, 'cause same effect can be achieved with `seek()` and `read()` methods

Reply:
I'd prefer we don't add new public methods to `Blob`, instead add this functionality to the `Client` class.